src/components/progress/progress.stories.tsx

```diff
@@ -41,9 +58,20 @@ export const Animation = () => {
                 return newValue;
             });
         }, 1000);
+
+        return () => clearInterval(intervalId);
     }, []);
 
-    return <Progress value={value} />;
+    return <Progress value={value} variant={variant} />;
+};
+
+/**
+ * Animation example of the Progress component with variant control.
+ */
+export const Animation: Story = {
+    render: ({ variant, value }: IProgressProps) => {
+        return <AnimatedProgress value={value} variant={variant} />;
+    },
```

Comment:
In that way we won't need to update this story if we add other properties
```suggestion
    render: ({ value, ...props }: IProgressProps) => <AnimatedProgress {...props} />,
```